src/components/pages/Users/EditUserPanel/EditUserPanel.tsx

```diff
@@ -241,13 +273,30 @@ const EditUserPanel = ({ close, user, userId, setPanelWidth }: Props) => {
               </ToastCard>
             ));
           }
-          if (!hasGroupsError && !hasRolesError && !hasEntitlementsError) {
+          if (hasUserError) {
+            reactHotToast.custom((t) => (
+              <ToastCard toastInstance={t} type="negative">
+                {Label.USER_ERROR}
+              </ToastCard>
+            ));
+          }
+          if (
+            !hasGroupsError &&
+            !hasRolesError &&
+            !hasEntitlementsError &&
+            !hasUserError
+          ) {
             reactHotToast.custom((t) => (
               <ToastCard toastInstance={t} type="positive">
                 {`User with email "${user.email}" was updated.`}
```

Comment:
This will need to use `values.email` as it might have changed from the saved user.